bachelors-workshop-game/scripts/player_character.gd

```diff
@@ -2,3 +2,65 @@ extends CharacterBody3D
 class_name PlayerCharacter
 
 @export var supply_count = 0
+const WALK_SPEED = 5.0
+const SPRINT_SPEED = 8.0
+const JUMP_VELOCITY = 4.5
+
+var sensitivity = 0.003
+var gravity = 9.8
+var speed
+var health = 100.0
+
+@onready var head = $Head
+@onready var camera = $Head/Camera3D
+
+func _ready():
+	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
+	
+func _input(event):
```

Comment:
Please remove this, managing game state (much less quitting the game) should not be up to the player controller